internal/cmds/cmds.go

```diff
@@ -295,6 +315,28 @@ func enable(ctx *log.Context, h types.HandlerEnvironment, report *types.RunComma
 	return stdoutTail, stderrTail, runErr, exitCode
 }
 
+func createExtensionEventManager(ctx *log.Context, hEnv types.HandlerEnvironment) *extensionevents.ExtensionEventManager {
+	el := logging.New(nil)
+	platformHandlerEnv := convertToPlatformHandlerEnv((hEnv))
+	extensionEvents := extensionevents.New(el, platformHandlerEnv)
+	return extensionEvents
+}
+
+func convertToPlatformHandlerEnv(myEnv types.HandlerEnvironment) *handlerenv.HandlerEnvironment {
+	data, err := json.Marshal(myEnv.HandlerEnvironment)
+	if err != nil {
+		return nil
```

Comment:
return error instead ?

Reply:
I originally was returning an error, but since extensionevents.New handles a null handler environment, and events are best effort, I simplified the code by just returning nil.